packages/core/botpress/src/web/views/Dashboard/index.jsx

```diff
@@ -57,8 +64,20 @@ export default class DashboardView extends React.Component {
     )
   }
 
+  get hero() {
```

Comment:
I'm not against getters but we're not using them anywhere else, instead we have the normal methods called `renderXYZ`. Is there any advantage of using the getter here?

Reply:
@emirotin no